reporter.js

```diff
@@ -3,8 +3,53 @@ var throughObj = require('through2').obj
 
 const PLUGIN_NAME = 'gulp-pug-linter'
 
-module.exports = function (hasToFail) {
+var defaultReporter = function (errors) {
+  if (errors.length) {
+    var allErrors = errors.map(function (error) {
```

Comment:
Please declare `var allErrors` at the top of this function, like so:

```js
var getDefaultReporter = function (errors) {
  var allErrors

  if (errors.length) {
    allErrors = // ...
```

Either way, `allErrors` is going to live within the scope of `defaultReporter`. For more information, [check out this StackOverflow answer](http://stackoverflow.com/a/11532092/545590).

---

This comment also applies to [the following line](https://github.com/ilyakam/gulp-pug-linter/pull/19/files#diff-fdfe8a4d956efc7b79e8d0c19bf2e0f6R18).